server/views/components/sample/sample.njk

```diff
@@ -1,6 +1,7 @@
 <div class="grid__cell">
   <div class="sample-component">
     <h2 class="sample-component__heading">Sample component</h2>
-    <p>And some sample text</p>
+    <p>{{ text }}</p>
+    {% include "views/components/grid-placeholder.njk" %}
   </div>
 </div>
```

Comment:
So we have a readme per unit? This is a lot of overhead no?
Or are we happy with that being the required amount of overhead per component.

In a way I like it as we can add metrics etc to them too - potentially even real times ones.